docs/ehrql-new-tutorial-conclusion.md

```diff
@@ -0,0 +1,48 @@
+# ehrQL tutorial: Conclusion
+
+---8<-- 'includes/data-builder-danger-header.md'
+
+## Summary
+
+Thank you for taking the time to work through this tutorial!
+We hope that you found it useful.
+
+!!! todo
+    Consider adding a call to action for feedback?
+
+    MkDocs for Material has options for this.
+    We could also consider just having a specific form.
+    Or simply requesting via the GitHub Discussions.
+
+At this point,
+you should have seen a number of concepts
+useful for writing dataset definitions:
+
+* What dataset definitions are
+* How to run dataset definitions on your own computer
+* What ehrQL is
+* How dataset definitions relate to running queries in OpenSAFELY backends
+* How to discover the data available in OpenSAFELY backends
+* The different kinds of tables we work with in ehrQL: patient-level and event-level, frames and series
+* The ways in which we can work with dates and handle missing values
+* Using medical codelists
+* How to run filtering and aggregation queries
+
+## Where to look next
+
+Now that you have completed this tutorial,
+you may find the following resources useful
+as reference for writing your own dataset definitions:
+
+* The [Contracts reference](contracts-reference.md)
+* The [ehrQL reference](ehrql-reference.md)
+* Existing dataset definitions in the `opensafely` GitHub organisation
```

Comment:
Link out to a couple of good examples, when we have them.

Reply:
Yep, I added a todo to that effect; hopefully we have a working study (or studies) in the main organisation soon.